src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -320,12 +320,6 @@ ConstObjectPtr SOP_SceneCacheSource::buildTransformPointCloud( const IECoreScene
 	}
 	std::vector<Imath::V3f> point( 1, p );
 	PointsPrimitivePtr points = new PointsPrimitive( new V3fVectorData( point ) );
-	std::vector<Imath::V3f> basis1( 1, Imath::V3f( transform[0][0], transform[0][1], transform[0][2] ) );
```

Comment:
When presenting something for review, it's preferable to avoid later commits that reverse the work of earlier commits. I've just reviewed the basis conversions and tests, and only now do I realise that I didn't need to.

Reply:
Indeed, that commit was later added based on Dan's review, I should have squashed it before pushing to Cortex public, my apologies for the trouble.